### ISSUE TITLE
ERR_SSL_PROTOCOL_ERROR on https://orchestrator.strn.pl and https://dashboard.saturn.tech

### ISSUE THREAD

**Issue by vorburger** [filecoin-saturn/L1-node#477]
### What did you do?

As per https://github.com/filecoin-saturn/L1-node#node-monitoring,

I tried to open both https://dashboard.saturn.tech/ and https://orchestrator.strn.pl/stats

### What happened?

Web browser displays: _This site can’t provide a secure connectionorchestrator.strn.pl sent an invalid response._ `ERR_SSL_PROTOCOL_ERROR` on the page (Orchestrator), respectively _GET https://orchestrator.strn.pl/stats?sortColumn=id net::ERR_SSL_PROTOCOL_ERROR_ on the Console (Dashboard).

### What would you expect to have happened?

See web page's content.

### How can we reproduce this issue?

I can reproduce this in x3 different web browsers, an (up-to-date) Firefox, Brave and Chromium.

This may be a temporary issue - are you just cycling certs right now, or something like that? I'm seeing it as of filing this issue.

**Comment by vorburger** [comment]
Ah, wait... I think this is something pretty serious... I've just noticed that my Saturn Node seems to fail to start, log says:

```
2023-08-27T13:41:42.814Z node@mainnet:registration Initiating registration (initial=true)
2023-08-27T13:41:42.904Z node@mainnet:registration Failed to fetch requirements: FetchError
2023-08-27T13:41:42.905Z node@mainnet Failed to register Error Failed to fetch requirements: FetchError
```

I suspect that's because https://orchestrator.strn.pl/requirements has the problem above...

**Comment by vorburger** [comment]
On https://filecoinproject.slack.com/archives/C03DH0BL02E/p1693144320041949 someone responded that _It loads fine for me from several locations in Europe._ I tried from several different devices, but all from the same location, and noticed that on one of my devices where I have a VPN it worked...

... so it Looks like it's broken from some locations, but not for others.

**Comment by vorburger** [comment]
Holy... f*% .. my ISP is blocking Saturn!!! 😭  Here's how I figured:

```bash
$ curl http://orchestrator.strn.pl/stats

<!doctype html>
<html>
    <head>
        <title>Website blocked by Swisscom Internet Guard</title>
        <meta charset="utf-8">
        <meta name="description" content="Swisscom Internet Guard protects you from accessing dangerous websites which may be involved in criminal activities.">

(...)
```

I'll check if there is a way to de-activate this _"protection from dangerous websites which may be involved in criminal activities"_

**Comment by vorburger** [comment]
So this is caused by [Swisscom Internet Guard](https://www.swisscom.ch/en/residential/internet/protection-security.html?campID=BI_GUARD_LINK_IGUARD#free=) ... 😞 

[This tool](https://www.swisscom.ch/en/residential/help/internet/internetguard/url-checker.html) allows to check (prove) this; for https://dashboard.saturn.tech it says _Not blacklisted_ but for https://orchestrator.strn.pl it's _Blacklisted: Blocked due to phishing!_ 😭 

For what it's worth (probably nothing), I have used [this form](https://www.swisscom.ch/en/residential/internet/protection-security/internet-guard-details/formular.html) to propose removing the blacklist entry - not sure if that will be succesful, but perhaps worth a try.

Accessing http://igtest.bluewin.org and http://blockphishing.bluewin.ch show customers of this ISP (Swisscom) if they are under Internet Guard tutelage.

BTW Google's Safe Browsing API does not flag the [dashboard.saturn.tech](https://transparencyreport.google.com/safe-browsing/search?url=dashboard.saturn.tech) or [orchestrator.strn.pl](https://transparencyreport.google.com/safe-browsing/search?url=orchestrator.strn.pl) these Saturn URLs.

I'm closing this issue because the maintainers of this project can't really "fix" this.

I have raised two separate new issue to follow-up with ideas related to / motivated by this experience - see #478 and #479.
